MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/AnimatedMarkerActivity.java

```diff
@@ -51,7 +51,7 @@ public void onMapReady(MapboxMap mapboxMap) {
           Toast.LENGTH_LONG
         ).show();
 
-        mapboxMap.setOnMapClickListener(new MapboxMap.OnMapClickListener() {
+        mapboxMap.addOnMapClickListener(new MapboxMap.OnMapClickListener() {
```

Comment:
if you add a listener it would also be cleaner to remove it as part of tearing the map object down with `MapboxMap#removeOnMapClickListener` to avoid leaking objects.

Reply:
Fixed

Reply:
Need to fix on other activities 🤦‍♂️